pandera/pandera.py

```diff
@@ -670,29 +807,40 @@ def _wrapper(fn, instance, args, kwargs):
 
 def check_output(schema, obj_getter=None):
     """Validate function output.
-
+    
     Similar to input validator, but validates the output of the decorated
     function. Note that the `transformer` function supplied to the
     DataFrameSchema will not have an effect in the check_output schema
     validator.
 
-    Parameters
-    ----------
-    schema : DataFrameSchema|SeriesSchema
-        dataframe/series schema object
-    obj_getter : int|str|callable|None
-        if int, assumes that the output of the decorated function is a
-        list-like object, where obj_getter is the index of the pandas data
-        dataframe/series to be validated. If str, expects that the output
-        is a dict-like object, and obj_getter is the key pointing to the
-        dataframe/series to be validated. If a callable is supplied, it expects
-        the output of decorated function and should return the dataframe/series
-        to be validated.
-
+    :param schema: dataframe/series schema object
+    :type schema: DataFrameSchema|SeriesSchema
+    :param obj_getter:  (Default value = None) if int, assumes that the output
+        of the decorated function is a list-like object, where obj_getter is
+        the index of the pandas data dataframe/series to be validated. If str,
+        expects that the output is a dict-like object, and obj_getter is the
+        key pointing to the dataframe/series to be validated. If a callable is
+        supplied, it expects the output of decorated function and should return
+        the dataframe/series to be validated.
+    :type obj_getter: int|str|callable|None
+
+    
     """
 
     @wrapt.decorator
     def _wrapper(fn, instance, args, kwargs):
+        """This function wrapper called each time a decorated function is
```

Comment:
I don't think private functions need docstrings, esp. if it conforms to an idiomatic design pattern

Reply:
removed